somef package/src/somef/configuration.py

```diff
@@ -16,6 +16,9 @@ def configure(authorization="",
         installation=default_installation,
         citation=default_citation):
 
+    import nltk
```

Comment:
Please add the imports on top. Otherwise is very difficult to follow dependencies.

Reply:
The reason for the import not being at the top is because nltk takes a long time to import. Previously, wordnet was installed separately, because we did not want to import nltk due to long import time. While this does not follow proper guidelines, I believe that automatically downloading wordnet is an improvement versus requiring that users manually install wordnet.